apps/dialtone-documentation/docs/components/avatar.md

```diff
@@ -16,6 +16,41 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
     </div>
 </code-well-header>
 
+<code-example-tabs
```

Comment:
If there are Variants and Examples on the same page, the first one doesn't need a code view. It mainly just serves as a quick visual. That's the pattern at play in current doc pages, e.g. https://dialtone.dialpad.com/components/avatar.html

Reply:
Sounds good, will remove.